### ISSUE TITLE
x/pkgsite: Alt text should be changed

### ISSUE THREAD

**Issue by ghost** [golang/go#42594]
This is a super minor issue, nothing urgent.  I noticed some alt-text on the go.dev homepage that could be improved.
 
### What is the URL of the page with the issue?
 https://go.dev/
 
 
### What is your user agent?
 

 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:82.0) Gecko/20100101 Firefox/82.0
 
 
### Screenshot

![image](https://user-images.githubusercontent.com/28990688/99107230-58cdb680-259a-11eb-80b1-f432efc9015a.png)

(I use a custom web font, which is why it looks different.  The font isn't relevant to the issue so I didn't bother disabling it.)
 

 
 
 
### What did you do?
 

 The page loaded and the arrows didn't load (I think they were blocked on my end or some packets were dropped, they worked fine when I reloaded), so I noticed the alt text.
 
 
### What did you expect to see?
 
 I figured that, because the images were blocked, they would be replaced with > instead of "navigate_next", because the latter hangs off of the button.
 
### What did you see instead?

I saw the current alt text, which hangs off of the button.
